src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/updateCVEDescription.hpp

```diff
@@ -46,57 +46,28 @@ class UpdateCVEDescription final
+            auto problemTypesArray = cna->problemTypes();
+            auto cve5Metadata = cve5Flatbuffer->cveMetadata();
 
             float VulnDescFBScoreBase = 0.0;
             std::string VulnDescFBClassificationStr;
             std::string VulnDescFBDescriptionStr;
             std::string VulnDescFBSeverityStr;
             std::string VulnDescFBScoreVersionStr;
             std::string VulnDescFBReferenceStr;
-
-            if (metricsArray)
-            {
-                for (const auto& field : *metricsArray)
-                {
-                    auto metricCVSSV3_1 = field->cvssV3_1();
-                    auto metricCVSSV3_0 = field->cvssV3_0();
-                    auto metricCVSSV2_0 = field->cvssV2_0();
-                    if (metricCVSSV3_1)
-                    {
-                        VulnDescFBScoreBase = metricCVSSV3_1->baseScore();
-                        auto baseSeverity = metricCVSSV3_1->baseSeverity();
-                        VulnDescFBSeverityStr = (nullptr != baseSeverity) ? baseSeverity->str() : "";
-                        auto version = metricCVSSV3_1->version();
-                        VulnDescFBScoreVersionStr = (nullptr != version) ? version->str() : "";
-                        auto format = field->format();
-                        VulnDescFBClassificationStr = (nullptr != format) ? format->str() : "";
-                        break;
-                    }
-                    else if (metricCVSSV3_0)
-                    {
-                        VulnDescFBScoreBase = metricCVSSV3_0->baseScore();
-                        auto baseSeverity = metricCVSSV3_0->baseSeverity();
-                        VulnDescFBSeverityStr = (nullptr != baseSeverity) ? baseSeverity->str() : "";
-                        auto version = metricCVSSV3_0->version();
-                        VulnDescFBScoreVersionStr = (nullptr != version) ? version->str() : "";
-                        auto format = field->format();
-                        VulnDescFBClassificationStr = (nullptr != format) ? format->str() : "";
-                        break;
-                    }
-                    else if (metricCVSSV2_0)
-                    {
-                        VulnDescFBScoreBase = metricCVSSV2_0->baseScore();
-                        VulnDescFBSeverityStr = (VulnDescFBScoreBase < 4.0)   ? "LOW"
-                                                : (VulnDescFBScoreBase < 7.0) ? "MEDIUM"
-                                                                              : "HIGH";
-                        auto version = metricCVSSV2_0->version();
-                        VulnDescFBScoreVersionStr = (nullptr != version) ? version->str() : "";
-                        auto format = field->format();
-                        VulnDescFBClassificationStr = (nullptr != format) ? format->str() : "";
-                        break;
-                    }
-                }
-            }
+            std::string VulnDescFBAssignerStr;
```

Comment:
Variable names should start in lowercase

Reply:
Changed